sparkmagic/sparkmagic/tests/test_kernel_magics.py

```diff
@@ -502,10 +502,50 @@ def test_spark_expected_exception():
     spark_controller.run_command = MagicMock(side_effect=SessionManagementException('oups'))
 
     magic.spark(line, cell)
-    spark_controller.run_command.assert_called_once_with(Command(cell))
+    spark_controller.run_command.assert_called_once_with(Command(cell), None)
     ipython_display.send_error.assert_called_once_with(constants.EXPECTED_ERROR_MSG
                                                        .format(spark_controller.run_command.side_effect))
 
+@with_setup(_setup, _teardown)
+def test_spark_expected_exception_in_storing():
+    line = "-o var_name"
+    cell = "some spark code"
+    side_effect = [(True,'ok'), Exception('oups')]
+    spark_controller.run_command = MagicMock(side_effect=side_effect)
+
+    magic.spark(line, cell)
+    assert spark_controller.run_command.call_count == 2
+    spark_controller.run_command.assert_any_call(Command(cell), None)
+    ipython_display.send_error.assert_called_with(constants.INTERNAL_ERROR_MSG
+                                                       .format(side_effect[1]))
+
+
+@with_setup(_setup, _teardown)
+def test_spark_expected_exception_in_storing():
+    line = "-o var_name"
```

Comment:
This test's name is the same as the one above. Please change the one above to unexpected.